src/components/cartPage/howDiscoverPicker/howDiscoverPicker.tsx

```
@@ -0,0 +1,112 @@
+              id="howDiscover"
+              type="text"
+              value={lang(pickedItem.name, language)}
+              className={st.field}
+            />
+          )}
+          {!pickedItem && (
+            <>
+              <span className={st.initialText}>
+                {lang('SelectFromList', language)}
+              </span>
+            </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)